app/src/main/res/values/strings.xml

```diff
@@ -70,7 +70,7 @@
     <string name="previous">Previous</string>
     <string name="hello_first_fragment">Hello first fragment</string>
     <string name="hello_second_fragment">Hello second fragment. Arg: %1$s</string>
-
+    <string name="show_friends">Show friends</string>
```

Comment:
Applying our naming convention, this should be profile_showFriends.

Reply:
There fixed it!